dubbo-common/src/test/java/org/apache/dubbo/config/context/ConfigManagerTest.java

```diff
@@ -267,7 +267,7 @@ public void testConfigMode() {
                 getConfigManager().addConfig(applicationConfig2);
                 fail("strict mode cannot add two application configs");
             } catch (Exception e) {
-                assertEquals(IllegalStateException.class, e.getClass());
```

Comment:
There should be no need to modify

Reply:
We should not theat that all kinds of exception are safe enough, some contains serialization threat.

Reply:
@AlbumenJ 
Sorry, I haven't pulled the latest code. The latest code has removed the write and read method. So I only modify the unstandard Indentation code in ConfigManager. Please check if the pr can be merged. Thank you.